components/common-go/cgroups/v2/cpu.go

```diff
@@ -43,7 +43,7 @@ func (c *Cpu) Max() (quota uint64, period uint64, err error) {
 	path := filepath.Join(c.path, "cpu.max")
 	content, err := os.ReadFile(path)
 	if err != nil {
-		return 0, 0, nil
+		return 0, 0, err
```

Comment:
@Furisto can you share how to test? It looks like the code path will [exit here](https://github.com/gitpod-io/gitpod/blob/91fa12d2edf22ef51b89e1a774d8deeb990f751b/components/ws-daemon/pkg/iws/iws.go#L1108) now, instead of trying to do the [division here](https://github.com/gitpod-io/gitpod/blob/91fa12d2edf22ef51b89e1a774d8deeb990f751b/components/ws-daemon/pkg/iws/iws.go#L1121). I ask because I'm not sure what scenario would have caused `os.ReadFile(path)` to return an error in the first place.